src/protocol/sort/mod.rs

```diff
@@ -1,9 +1,9 @@
+pub mod apply;
```

Comment:
see above, we should consider remove this from sort and add it e.g. as a basic protocol